spec/components/access_panels/in_collection_component_spec.rb

```diff
@@ -16,4 +16,20 @@
       expect(page).to have_no_content("Digital content")
     end
   end
+
+  describe 'merged records with collection members' do
```

Comment:
would you mind updating this to reflect that we're testing escaped characters in the summary field?

Reply:
not at all. good catch.